AppKit/CPSplitView.j

```diff
@@ -1205,6 +1186,32 @@ The sum of the views and the sum of the dividers should be equal to the size of
 
 @end
 
+@implementation CPSplitView (CPTrackingArea)
+
+- (void)updateTrackingAreas
+{
+    var myTrackingAreas = [self trackingAreas];
+    
+    for (var i = 0; i < myTrackingAreas.length; i++)
+        [self removeTrackingArea:myTrackingAreas[i]];
```

Comment:
It would be cleaner and faster to have a method called _removeAllTrackingAreas in CPView where you empty the array and do the cleanup. Here, you are creating an copy of an array and then emptying the array based on its copy. It seems this new method can be used in other subclasses that need an -updateTrackingAreas.